test/oComments.test.js

```diff
@@ -62,6 +74,38 @@ describe("OComments", () => {
 		});
 	});
 
+	describe("when an authentication fails", () => {
+		const sandbox = sinon.sandbox.create();
+		let comments;
+		let rootElement;
+		let scriptElement;
+		let oCommentsReadyListener;
+
+		before((done) => {
+			sandbox.stub(auth, 'getJsonWebToken').returns(Promise.reject(new Error()));
```

Comment:
Related to the previous
```suggestion
			sandbox.stub(auth, 'getJsonWebToken').rejects(new Error());
```